server/cluster/cluster.go

```diff
@@ -2074,7 +2074,8 @@ func (c *RaftCluster) AddStoreLimit(store *metapb.Store) {
 		}
 	}
 
-	cfg.StoreLimit[storeID] = sc
+	id := strconv.FormatUint(storeID, 10)
```

Comment:
move it before line 2062 so that it won't be called twice

Reply:
Good catch! Done.